### ISSUE TITLE
Cluster.ready? returns false 

### ISSUE THREAD

**Issue by villemuittari** [fog/fog-aws#496]
When I start a cluster and check if it's ready, it always returns false. Cluster.state seems to have value "active" instead of "available". This leads Cluster.ready? to return false all the time.

https://github.com/fog/fog-aws/blob/c831099060bfa35c02fd4b8ee3ecb52e323007d0/lib/fog/aws/models/rds/cluster.rb#L26

https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Status.html

**Comment by geemus** [comment]
Thanks for the heads up, probably just a copypasta error.

Would you like to help us out with a patch? Thanks!

**Comment by villemuittari** [comment]
Following should explain the problem. 

```
cluster = rds.clusters.get('db-staging-cluster')
puts(cluster.ready?)
puts(cluster.inspect)
```

Result:
```
false
<Fog::AWS::RDS::Cluster
    id="db-staging-cluster",
    allocated_storage=1,
    backup_retention_period=10,
    db_cluster_members=[{"DBInstanceIdentifier"=>"db-staging", "master"=>true}],
    db_cluster_parameter_group="default.aurora5.6",
    db_subnet_group=nil,
    engine="aurora",
    engine_version="5.6.10a",
    port=3306,
    preferred_backup_window="09:00-09:30",
    preferred_maintenance_window="sun:04:30-sun:05:00",
    state="active",
    vpc_security_groups=[{"VpcSecurityGroupId"=>"xxxxxx"}]
  >
```

```
server = rds.servers.get('db-staging')
puts(server.ready?)
puts(server.inspect)
```  

Result:
```
<Fog::AWS::RDS::Server
    id="db-staging",
    allocated_storage=1,
    auto_minor_version_upgrade=true,
    availability_zone="us-east-1a",
    backup_retention_period=10,
    ca_certificate_id="rds-ca-2015",
    character_set_name=nil,
    cluster_id="db-staging-cluster",
    created_at=2018-11-28 08:16:01 UTC,
    db_name="xxxxx",
    db_parameter_groups=[{"DBParameterGroupName"=>"default.aurora5.6", "ParameterApplyStatus"=>"in-sync"}],
    db_security_groups=[],
    db_subnet_group_name="subnet-group-one",
    engine="aurora",
    engine_version="5.6.10a",
    flavor_id="db.t2.small",
    iops=nil,
    kms_key_id=nil,
    last_restorable_time=nil,
    license_model="general-public-license",
    multi_az=false,
    pending_modified_values={},
    preferred_backup_window="09:00-09:30",
    preferred_maintenance_window="thu:09:51-thu:10:21",
    publicly_accessible=false,
    read_replica_identifiers=[],
    read_replica_source=nil,
    state="available",
    storage_encrypted=false,
    storage_type="aurora",
    tde_credential_arn=nil,
    vpc_security_groups=[{"VpcSecurityGroupId"=>"xxxxxx", "Status"=>"active"}]
  >
```

**Comment by geemus** [comment]
Sure, I understood. I just thought you might like to make the patch and get credit. Not a big deal either way, so just let me know if you would prefer me to do it.

**Comment by geemus** [comment]
Fixed by #497
